code_annotations/config_and_tools/config/feature_toggle_annotations.yaml

```diff
@@ -11,17 +11,17 @@ annotations:
   feature_toggle:
     - ".. toggle_name:":
     - ".. toggle_implementation:":
-        choices: [ExperimentWaffleFlag, WaffleFlag, WaffleSample, WaffleSwitch, CourseWaffleFlag, ConfigurationModel, DjangoSetting]
+        choices: [WaffleFlag, WaffleSwitch, CourseWaffleFlag, ExperimentWaffleFlag, ConfigurationModel, DjangoSetting]
     - ".. toggle_default:":
     - ".. toggle_description:":
-    - ".. toggle_category:":
     - ".. toggle_use_cases:":
-        choices: [incremental_release, launch_date, monitored_rollout, graceful_degradation, beta_testing, vip, opt_out, opt_in, open_edx]
+        choices: [temporary, circuit_breaker, vip, opt_out, opt_in, open_edx]
     - ".. toggle_creation_date:":
     - ".. toggle_expiration_date:":
```

Comment:
Since this is intended to be a sustainable pipeline into the DEPR process, it uses the phrase ["Target Removal Date"](https://open-edx-proposals.readthedocs.io/en/latest/oep-0021-proc-deprecation.html#to-the-open-edx-discourse).  We can do the same here, yeah?

`toggle_target_removal_date`

Reply:
:ok_hand: